### DISCUSSION TITLE
Creating new theme failed

### DISCUSSION THREAD

**Discussion by yyyyj** [requarks/wiki#6575] [Help / Questions]
Good morning,

I was trying to make a new theme as described in this page : https://docs.requarks.io/dev/themes. However, when I switch the theming value to another value (e.g. "altes" in my example) the loader doesn't find any files. Is there a way to solve that issue ? 

I'm using Windows 11 + Docker, DBeaver with the PostGre SQL db with a clone of the Wiki.js github repository.
![ScreenDBeaver](https://github.com/requarks/wiki/assets/26669720/a370447f-e40b-46b3-aff6-12b3c40a0d06)
![vscode](https://github.com/requarks/wiki/assets/26669720/5bd302e1-2539-41ec-9fec-6cce574ba209)
![screen_firefox](https://github.com/requarks/wiki/assets/26669720/6b845b88-19f3-4f27-99d4-c1f6a84a7e67)
The altes folder is, for now, a pure copy of default one.

**Comment by NGPixel** [comment]
You need to rebuild the client assets. Simply duplicating the folder in the assets folder won't do anything.
